
# Publish all joint states -----------------------------------
joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50
  
front_left_wheel_axel_controller:
    type: velocity_controllers/JointVelocityController
    joint: front_left_wheel_axel
    pid: {p: 100.0, i: 0.01, d: 10.0}
back_left_wheel_axel_controller:
    type: velocity_controllers/JointVelocityController
    joint: back_left_wheel_axel
    pid: {p: 100.0, i: 0.01, d: 10.0}
front_right_wheel_axel_controller:
    type: velocity_controllers/JointVelocityController
    joint: front_right_wheel_axel
    pid: {p: 100.0, i: 0.01, d: 10.0}
back_right_wheel_axel_controller:
    type: velocity_controllers/JointVelocityController
    joint: back_right_wheel_axel
    pid: {p: 100.0, i: 0.01, d: 10.0}
    
front_left_swerve_controller:
    type: position_controllers/JointPositionController
    joint: front_left_swerve
    pid: {p: 100.0, i: 0.01, d: 10.0}
back_left_swerve_controller:
    type: position_controllers/JointPositionController
    joint: back_left_swerve
    pid: {p: 100.0, i: 0.01, d: 10.0}
front_right_swerve_controller:
    type: position_controllers/JointPositionController
    joint: front_right_swerve
    pid: {p: 100.0, i: 0.01, d: 10.0}
back_right_swerve_controller:
    type: position_controllers/JointPositionController
    joint: back_right_swerve
    pid: {p: 100.0, i: 0.01, d: 10.0}
    
laser_tilt_joint_controller:
    type: position_controllers/JointPositionController
    joint: laser_tilt_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}
